packages/eas-cli/src/build/ios/bundleIdentifer.ts

```diff
@@ -7,7 +7,11 @@ import { promptAsync } from '../../prompts';
 
 export const getBundleIdentifier = once(_getBundleIdentifier);
 
-async function _getBundleIdentifier(projectDir: string, manifest: ExpoConfig): Promise<string> {
+async function _getBundleIdentifier(
+  projectDir: string,
+  manifest: ExpoConfig,
+  displayAutoconfigMessage = true
```

Comment:
I'd slightly prefer to convert the boolean argument to an object:
```suggestion
  { displayAutoconfigMessage = true }: { displayAutoconfigMessage?: boolean }
```
When reading code on the call-site, the only thing you see is `getBundleIdentifier(..., ... false)`. And you can't really figure out what the `false` argument means. `getBundleIdentifier(..., ... { displayAutoconfigMessage: true })` is straightforward.

Reply:
^This comment made me missing old DirectX/WinApi calls like `SomeWinApiCall(handle, NULL, NULL, 0, NULL, 0, 0, true)` 😅